metadata.json

```diff
@@ -7,34 +7,34 @@
   "source": "https://github.com/mlibrary/nebula",
   "issues_url": "https://github.com/mlibrary/nebula/issues",
   "dependencies": [
-    {"name": "jdowning/rbenv",          "version_requirement": ">= 3.0.0  < 4.0.0"},
-    {"name": "puppet/archive",          "version_requirement": ">= 6.1.2  < 7.0.0"},
-    {"name": "puppet/kmod",             "version_requirement": ">= 3.2.0  < 4.0.0"},
-    {"name": "puppet/letsencrypt",      "version_requirement": ">= 9.2.0  < 10.0.0"},
-    {"name": "puppet/logrotate",        "version_requirement": ">= 6.1.0  < 7.0.0"},
-    {"name": "puppet/nginx",            "version_requirement": ">= 4.4.0  < 5.0.0"},
-    {"name": "puppet/php",              "version_requirement": ">= 8.2.0  < 9.0.0"},
-    {"name": "puppetlabs/apache",       "version_requirement": ">= 7.0.0  < 8.0.0"},
-    {"name": "puppetlabs/apt",          "version_requirement": ">= 8.5.0  < 9.0.0"},
-    {"name": "puppetlabs/augeas_core",  "version_requirement": ">= 1.4.0  < 2.0.0"},
-    {"name": "puppetlabs/concat",       "version_requirement": ">= 7.4.0  < 8.0.0"},
-    {"name": "puppetlabs/cron_core",    "version_requirement": ">= 1.2.0  < 2.0.0"},
-    {"name": "puppetlabs/docker",       "version_requirement": ">= 6.1.0  < 7.0.0"},
-    {"name": "puppetlabs/firewall",     "version_requirement": ">= 4.1.0  < 5.0.0"},
-    {"name": "puppetlabs/host_core",    "version_requirement": ">= 1.2.0  < 2.0.0"},
-    {"name": "puppetlabs/inifile",      "version_requirement": ">= 5.4.1  < 6.0.0"},
-    {"name": "puppetlabs/java",         "version_requirement": ">= 9.0.1  < 10.0.0"},
-    {"name": "puppetlabs/lvm",          "version_requirement": ">= 1.4.0  < 2.0.0"},
-    {"name": "puppetlabs/mount_core",   "version_requirement": ">= 1.2.0  < 2.0.0"},
-    {"name": "puppetlabs/mysql",        "version_requirement": ">= 13.3.0 < 14.0.0"},
-    {"name": "puppetlabs/ntp",          "version_requirement": ">= 9.2.2  < 10.0.0"},
-    {"name": "puppetlabs/postgresql",   "version_requirement": ">= 8.3.0  < 9.0.0"},
-    {"name": "puppetlabs/puppetdb",     "version_requirement": ">= 7.13.0 < 8.0.0"},
-    {"name": "puppetlabs/reboot",       "version_requirement": ">= 4.3.1  < 5.0.0"},
-    {"name": "puppetlabs/sshkeys_core", "version_requirement": ">= 2.4.0  < 3.0.0"},
-    {"name": "puppetlabs/stdlib",       "version_requirement": ">= 8.6.0  < 9.0.0"},
-    {"name": "saz/resolv_conf",         "version_requirement": ">= 5.1.0  < 6.0.0"},
-    {"name": "stm/debconf",             "version_requirement": ">= 5.0.0  < 6.0.0"}
+    {"name": "jdowning/rbenv",          "version_requirement": "3.0.0"},
```

Comment:
I'm not entirely clear on the change from optimistic ranges to exact values. The modules have been very reliable in flagging compatibility issues with major versions and this change would eliminate any automatic updates we are currently receiving. I do like the synchronization with the fixtures.

Have you outlined any thoughts on inventory/updates with exact versions?

Reply:
I've been bitten by breaking changes introduced in modules on minor or teeny versions in the last two days. apt 9.0.1 doesn't even compile unless stdlib is updated to a version newer than what we're using and *much* newer than the version it claims to support.

Reply:
TBH this change was half due to the problems with puppetlabs/apt and half because the syntax was tedious (no `~>`!), which isn't a great reason. I also didn't realize that librarian-puppet was actually optimistically grabbing the newest rather than oldest compatible version.

I'll limit this stricture to where there is a known breaking change.